### PR TITLE
Add files for cs221 ch04s06 ge

### PR DESCRIPTION
**Pull request by niroy**

### CODE REVIEW

File: resources/observability-app-metrics/grafana-config.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+serviceAccount:
```
Comment (herve4m) [inline resources/observability-app-metrics/grafana-config.yaml:2]:
```
$ yamllint *.yaml
grafana-config.yaml
  1:1       warning  missing document start "---"  (document-start)
```

File: resources/observability-app-metrics/grafana-config.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  annotations:
+    eks.amazonaws.com/role-arn: <CHANGE_ME>
+securityContext:
+  runAsUser: null
+  runAsGroup: null
+  fsGroup: null
+  runAsNonRoot: true
+  seccompProfile:
+    type: RuntimeDefault
+
```
Comment (herve4m) [inline resources/observability-app-metrics/grafana-config.yaml:14]:
```suggestion
```

**Review by herve4m — APPROVED** [review]
LGTM

File: resources/observability-app-metrics/grafana-config.yaml (hunk 1 above)
Comment (niroy) [reply resources/observability-app-metrics/grafana-config.yaml:2]: Fixed.

File: resources/observability-app-metrics/grafana-config.yaml (hunk 2 above)
Comment (niroy) [reply resources/observability-app-metrics/grafana-config.yaml:14]: Fixed.

**Pull request merged**